syntax = "proto3";

package order;

// Message for OrderBase
message OrderBase {
    string order_address = 1;
    string phone_number = 2;
    float total_price = 3;
    string order_payment = 4;
}

// Message for OrderItemForm
message OrderItemForm {
    string product_id = 1;
    int32 product_item_id = 2;
    int32 product_size_id = 3;
    int32 quantity = 4;
}

// Message for OrderModel
message OrderModel {
    int32 user_id = 1;
    string order_id = 2;
    OrderBase base = 3;
    repeated OrderItemForm items = 4;
}
